spec/liquid_tags/liquid_tag_base_spec.rb

```diff
@@ -12,8 +12,8 @@
       source = create(:comment)
       expect do
         liquid_tag_options = { source: source, user: source.user }
-        Liquid::Template.parse("{% liquid_tag_base %}", liquid_tag_options)
-      end.to raise_error(LiquidTags::Errors::InvalidParseContext)
+        Liquid::Template.parse("{% liquid_tag_base %}", liquid_tag_options).errors
+      end.to raise_error(TypeError)
```

Comment:
this should go back to `LiquidTags::Errors::InvalidParseContext`, the reason why it's raising:

```shell
     TypeError:
       exception class/object expected
```

it's because `LiquidTags::Errors::InvalidParseContext` is a regular class, not an exception